src/App.jsx

```diff
@@ -14,70 +17,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
```

Comment:
Would be great to stay strict with the task description
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/c5331d50-c87f-4140-aa87-335ae8f1fa10)